src/types.rs

```diff
@@ -168,3 +176,14 @@ where
 		sig.verify(&*msg, &self.part)
 	}
 }
+
+impl<ChannelId, PK> Funding<ChannelId, PK>
+where
+	Funding<ChannelId, PK>: Encode,
+{
+	/// Calculates the funding id of a participant in a channel.
+	pub fn id<H: Hasher>(&self) -> H::Out {
```

Comment:
Where and how is the funding type used? I don't see any usages in this commit.

Reply:
It is only introduced in that commit, the transition from `calc_funding_id` to `Funding.id()` happens in the next commit.